code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -513,42 +513,44 @@
 		return TRUE
 	return FALSE
 
+/mob/living/simple_animal/bot/medbot/proc/drop_medkit(drop_turf)
+
+	switch(skin)
+		if("ointment")
+			new /obj/item/storage/firstaid/fire/empty(drop_turf)
+		if("tox")
+			new /obj/item/storage/firstaid/toxin/empty(drop_turf)
+		if("o2")
+			new /obj/item/storage/firstaid/o2/empty(drop_turf)
+		if("brute")
+			new /obj/item/storage/firstaid/brute/empty(drop_turf)
+		if("adv")
+			new /obj/item/storage/firstaid/adv/empty(drop_turf)
+		if("bezerk")
+			new /obj/item/storage/firstaid/tactical/empty(drop_turf)
```

Comment:
```suggestion
			new /obj/item/storage/firstaid/fake_tactical(drop_turf)
```
Make sure you use the craftable version when creating and deconstructing the medbots. It's most likely why you can't manage to make non-syndicate suspicious medbots.

Reply:
also, make sure this doesn't break actual syndicate medbot dropping the right kit.

Reply:
Syndicate medbots don't actually drop anything under any circumstance IIRC. And the "fake_syndicate" might be why my medbots are suddenly trying to choke me out again. Wasn't there when I started the PR

Reply:
I'll need to add a check to differentiate fake and real tactical medkits for that.